snap/info_snap_yaml.go

```diff
@@ -53,9 +60,23 @@ type slotYaml struct {
 }
 
 type appYaml struct {
-	Command   string   `yaml:"command"`
-	SlotNames []string `yaml:"slots,omitempty"`
-	PlugNames []string `yaml:"plugs,omitempty"`
+	Command string `yaml:"command"`
+
+	Daemon string `yaml:"daemon"`
+
+	Stop        string          `yaml:"stop-command,omitempty"`
+	PostStop    string          `yaml:"post-stop-command,omitempty"`
+	StopTimeout timeout.Timeout `yaml:"stop-timeout,omitempty"`
+
+	RestartCond systemd.RestartCondition `yaml:"restart-condition,omitempty"`
+	SlotNames   []string                 `yaml:"slots,omitempty"`
+	PlugNames   []string                 `yaml:"plugs,omitempty"`
+
+	BusName string `yaml:"bus-name,omitempty"`
```

Comment:
Thinking about this, I wonder what's the user. In the past it was used for frameworks but now that's entirely under the control of interfaces. I have a feeling that we will end up removing this field. At the same time it has some interplay with systemd units. Hmm 

CC @jdstrand